### PR TITLE
Phase2-hgx131 Add a new scenario with HFNose

### PR DESCRIPTION
**Pull request by bsunanda**

### CODE REVIEW

**Comment by bsunanda** [comment]
@cmsbuild Please test

File: Configuration/Geometry/python/dict2023Geometry.py [hunk 1]
```diff
@@ -562,7 +562,7 @@
         1 : [
             'Geometry/ForwardCommonData/data/forwardshield/2017/v1/forwardshield.xml',
             'Geometry/ForwardCommonData/data/brmrotations.xml',
-            'Geometry/ForwardCommonData/data/PostLS2/brm.xml',
+            'Geometry/ForwardCommonData/data/brm/2023/v1/brm.xml',
```
Comment (kpedro88) [inline Configuration/Geometry/python/dict2023Geometry.py]: @bsunanda why did you make this change in F2? This will affect all the Phase2 geometries.

**Comment by davidlange6** [comment]
please test

**Comment by bsunanda** [comment]
@cmsbuild Please test

**Comment by kpedro88** [comment]
+upgrade
(I privately tested new WF 26234.0, including #23810 for the topology class, and it runs successfully.)

**Comment by kpedro88** [comment]
@prebello @fabozzi please sign

**Comment by prebello** [comment]
+1

**Comment by kpedro88** [comment]
+1

**Comment by kpedro88** [comment]
merge

**Pull request merged**